benchmarks/python/conftest.py

```diff
@@ -98,6 +98,10 @@ def pytest_configure(config):
 
     if config.getoption("--benchmark-num-inputs"):
         BENCHMARK_CONFIG["num_inputs"] = int(config.getoption("--benchmark-num-inputs"))
+
+    # Scheduler markers were determined by running the benchmarks and aggregating logs with
+    # NVFUSER_DUMP=segmented_fusion. This was done this per-file, so for a file with multiple
+    # benchmarks inside, the individual benchmarks may have markers that don't apply.
```

Comment:
I believe you modified the markers to be per-benchmark per-param, so you can update this comment now.